apps/acrs/index.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
       behaviours: [SummaryPageBehaviour, CheckInformationGivenBehaviour],
       sections: require('./sections/summary-data-sections'),
       backLink: false,
-      journeyStart: '/who-is-completing-form'
+      journeyStart: '/who-completing-form'
```

Comment:
I think it needs to be `next: '/who-completing-form'` else it won't move on from this page

Reply:
It navigate and working fine with journeyStart. If you change it to next, you will need to modify continue-report.js which will break most of the code

Reply:
Yeah all good. Have approved